vm/devices/storage/disk_nvme/nvme_driver/src/namespace.rs

```diff
@@ -150,19 +148,15 @@ impl Namespace {
             resize_event: Default::default(),
         });
 
-        // Spawn a task, but detach is so that it doesn't get dropped while NVMe
-        // request is in flight. Use a cancel context, whose cancel gets dropped
-        // when `self` gets dropped, so that it terminates after finishing any
-        // requests.
-        let (mut ctx, cancel_rescan) = CancelContext::new().with_cancel();
+        // NOTE: Because `poll_for_rescans` exits when the sender for
```

Comment:
I think i would clarify this that the task itself will terminate, but we still _detach_ the task from self.